.github/workflows/release.yml

```diff
@@ -3,60 +3,109 @@ name: Release
 on:
   workflow_dispatch:
     inputs:
-      deploy-to:
-        type: choice
-        description: Choose where to publish (test/prod)
-        options:
-        - prod
-        - test
-        default: prod
-      ref:
-        description: "The ref (sha or branch name) to use"
+      branch:
+        description: "The branch to release from"
         type: string
         default: "main"
+      version:
+        description: "The version to release"
         required: true
+        type: string
+      deploy-to:
+        description: "Deploy to test or prod"
+        type: environment
+        default: prod
```

Comment:
Should this be a drop down? https://tkolber.medium.com/dropdown-input-choice-workflows-in-github-actions-e95cf140bf61

Reply:
I believe the type `environment` does show up in the UI as a drop down. It just avoids the need to explicitly list the environments configured in the repo.